src/components/DownloadToast/utils.ts

```diff
@@ -0,0 +1,5 @@
+import * as fileSize from 'filesize'
+
+export const size = fileSize.partial({ base: 10, round: 1, spacer: '' }) as (
+  arg: unknown
+) => string
```

Comment:
Can we use https://github.com/HyperPlay-Gaming/utils/blob/main/src/parseNumIntoReadableString.ts instead? It is the same function we use everywhere else, it has full test coverage, and doesn't introduce another dependency.

Reply:
We use `filesize` on the client already which won't duplicate the dependency when using there.
Another idea would be to avoid any calculation on this component and make it deal with the UI only, I think that should be better since those UI component should not really deal with any logic, only receive the info and render it.
What do you think?

Reply:
done, changed for the utils method